runtime/doc/options.txt

```diff
@@ -49,9 +49,12 @@ achieve special effects.  These options come in three forms:
 :se[t] {option}&vi	Reset option to its Vi default value.
 :se[t] {option}&vim	Reset option to its Vim default value.
 
-:se[t] all&		Set all options, except terminal options, to their
-			default value.  The values of 'lines' and 'columns'
-			are not changed.
+:se[t] all&		Set all options to their default value.  The values of
+			these options are not changed:
+			  'columns'
+			  'encoding'
+			  'lines'
```

Comment:
Thanks for noticing this. Though even 'encoding' is not worth mentioning for us, but we will address that en-masse eventually.